.github/workflows/release-tag.yml

```diff
@@ -0,0 +1,28 @@
+name: Create release tag
+
+on: create
```

Comment:
And only on branches named `release/**`

Reply:
Hmm, that is going to be really annoying tbh.

I assume creating a new branch from github does not create a "push" event? Otherwise we could have used that..

We need to also add the on push to release branches btw, since we can create release branches locally and push them, that might be the only workaround I foresee right now.

Maybe we additionally can use the `if` condition as mentioned in that thread..

Reply:
I tested the push event and managed to come close to what we need https://github.com/wizlif/test_git_actions/actions